src/index.html

```diff
@@ -7,6 +7,308 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+
+      <div class="header__logo"></div>
+
+      <nav class="navigation">
+        <ul class="navigation__items-list">
+          <li class="navigation__items-list__item">
+            <a href="#" class="navigation__link is-active">Apple</a>
```

Comment:
class `is-active` should be connected with `navigation__link` as `navigation__link--is-active`, it shouldn't be as a separate class, [see here](https://mate-academy.github.io/style-guides/htmlcss.html#bem-modifier-usage)